pig/src/main/java/org/apache/zeppelin/pig/PigInterpreter.java

```diff
@@ -58,6 +59,12 @@ public void open() {
     }
     try {
       pigServer = new PigServer(execType);
+      for (Map.Entry entry : getProperty().entrySet()) {
+        if (!entry.getKey().toString().startsWith("zeppelin.")) {
+          pigServer.getPigContext().getProperties().setProperty(entry.getKey().toString(),
+              entry.getValue().toString());
```

Comment:
As far as I recall, this is actually somewhat opposite to how we pass config to other interpreters.
Generally, we say, if it's spark, all property starting with "spark." we will pass through.
Here we are say if it's _not_ "pig." pass through